functions/package.json

```diff
@@ -10,9 +12,29 @@
     "node": "14"
   },
   "bin": {
-    "import": "./src/bin/import.js"
+    "app-search-firestore-extension": "./lib/bin/import.js"
   },
+  "directories": {
+    "lib": "lib"
```

Comment:
I didn't need to add this, I just did for completeness.

Reply:
Can you elaborate a little?

Reply:
I mean we don't need this to run or publish, it's just meta data that is sometimes useful to have: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#directories